__tests__/app.test.js

```diff
@@ -115,7 +97,7 @@ describe("/api/articles/:article_id", () => {
         .get("/api/articles/5000")
         .expect(404)
         .then((res) => {
-          const message = res.body.msg
+          const message = res.body.msg;
           expect(message).toBe("article not found!");
         });
```

Comment:
I would add a further test for a 400 - bad request e.g. a user tries to find an article with an id of abc (not a number)